src/components/Map/Map.tsx

```diff
@@ -64,20 +64,57 @@ function ConfigureMap({ center, zoom }: MapConfigProps) {
   return null;
 }
 
+// function Form() {
```

Comment:
I think we can go ahead and delete this Form Component, we went a different way and worked the form into the Map component instead.  Just clean-up, nothing more.